content/index.html

```diff
@@ -118,6 +118,7 @@ <h1><i class="fa fa-github"></i> Open Source</h1>
       <div class="row logos">
           <a href="https://amadeus.com/"><img src="assets/company-logos/Amadeus.png"/></a>
           <a href="https://argus-sec.com/"><img src="assets/company-logos/Argus.png"/></a>
+          <a href="https://www.ayedo.de/"><img src="assets/company-logos/ayedo.png"></a>
```

Comment:
The link is correct, but the image file will need to be added under the `company-logos` directory, not the `commercial-support-logos`. Currently it's a broken image in the preview deploy.

Reply:
Thanks @juliusv for the feedback and prompt response. My bad and updated it :)